tests/testthat/test-filter-dm.R

```diff
@@ -97,42 +97,46 @@ test_that("we get filtered/unfiltered tables with respective funs", {
   )
 })
 
+test_that("dm_filter() works on different srcs", {
+  expect_identical(
+    dm_filter(dm_for_filter, t1, a < 8, a > 3) %>% dm_apply_filters() %>% dm_get_tables(),
+    output_1
+  )
 
-
+  # FIXME: maybe always include this test on a DB, despite PR #313?
```

Comment:
Why?

Reply:
Idea was, that the things happening to the `dm` object (or maybe rather, how they are happening) depend to a certain extent on the `src` it is located. These differences are handled by {dbplyr}, so there is probably no real need to force the testing to be executed on a DB. I am gonna be more conservative from now on.